src/core/core-adapter.js

```diff
@@ -325,24 +328,32 @@ export default class CoreAdapter {
         // querySource: this.globalStorage.getState(StorageKeys.QUERY_SOURCE)
       })
       .then(data => {
+        const universalResults = adaptUniversalSearchResponse(data, urls);
+        const navigation = Navigation.fromVerticalResults(data.verticalResults);
+        const directAnswer = adaptDirectAnswer(data.directAnswer);
+        const searchIntents = SearchIntents.from(data.searchIntents);
+        const spellCheck = SpellCheck.from(data.spellCheck);
+        const locationBias = adaptLocationBias(data.locationBias);
```

Comment:
This is a stylistic call so up to you, what are your thoughts on the pattern where we have an adapter like the current `.then(response => SearchDataTransformer.transformVertical(response, this._fieldFormatters, verticalKey))` that turns an adapted object with this data vs declaring these constants?

Reply:
ya, that makes sense